packages/shared/src/dropdown/singleselectdropdown.tsx

```diff
@@ -30,6 +39,12 @@ export const SingleSelectDropdown: React.FC<SingleSelectDropdownProps> = ({
     [valueLabelMap, onChange, setOpen]
   );
 
+  const onCreateOption = (newValue: string) =>
+    setNewOptions([
+      ...newOptions,
+      <SelectOption key={newValue} value={newValue} />,
+    ]);
+
   return (
     // surround select with data-test-id to be able to find it in tests
     <div className="test" data-test-id={props['data-test-id']}>
```

Comment:
`data-test-id` is legacy convention...
```suggestion
    <div className="test" data-test={props['data-test']}>
```